docs/Makefile

```
@@ -3,17 +3,23 @@
 
 # You can set these variables from the command line.
 PYTHON        = python3
-SPHINXOPTS    =
 SPHINXBUILD   = $(PYTHON) -m sphinx.cmd.build
-PAPER         =
+SPHINXOPTS    = --fail-on-warning --keep-going
```

```suggestion
SPHINXOPTS    = --fail-on-warning
```

https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-keep-going
> From Sphinx 8.1, --keep-going is always enabled.
> ...
> Changed in version 8.1: sphinx-build no longer exits on the first warning, meaning that in effect --fail-on-warning is always enabled. The option is retained for compatibility, but may be removed at some later date.